### PR TITLE
reviewing-code

### PR DESCRIPTION
**Pull request by rivasbolinga**
Reviewing the code of the trials for Microverse

### CODE REVIEW

**Review by rivasbolinga — COMMENTED** [review]
Hi @rivasbolinga Congratulations! you have done an amazing job so far. But there are still some issues that you can improve, but you are certainly almost there!

**Highlights 💯**

- No linters errors ✔️
- Use of HTML and CSS best practices ✅
- Good use of GitHub Flow ✅

 **Required changes**
_Check the comments under the review._
-I think you should add a Descriptive README file. You can use this template https://github.com/microverseinc/readme-template/blob/master/README.md, but remember to customize it to your project.
-I recommend you to improve your commit messages, write descriptive comments professionally. Each commit should be small enough to describe it in one sentence. The commit message should be written in the imperative tense and capitalized.

**Review by sajid-munawar — CHANGES_REQUESTED** [review]
# Changes Required
 ### Hi @rivasbolinga  ,
 Nice job, you are almost there.
 
 

 
 ## (Changes Required) Aspects to improve: 
 * [ ] Kindly add descriptive inline comments. this way you can let the student understand that  why the request changes are asked. [for more info ](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md#how-to-communicate)
 * [ ] Nice pointing out the requested changes but Kindly make sure that you left the comment under a line or block of code that fits the context the comment is about.Please use the [GitHub inline comments feature](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request).



 ## [Optional] suggestions:
 * Nothing to mention.
 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
 **Cheers and Happy coding! :clap::clap::clap:**
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
> Please, remember to tag @sajid-munawarin your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @rivasbolinga  :wave: 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some changes you still need to work on before you move on to the next project but you are almost there.



 ## Highlights :+1: 
:heavy_check_mark: Good to highlight what you did well in your project through review.


 ## Required Changes ♻️
_Check the comments under the review._

- [ ] As requested by the previous code reviewer you should kindly add descriptive inline comments. This will let the person you reviewed his/her code know where and why the changes are requested.[For more info](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md#how-to-communicate). 
Please use the [GitHub inline comments feature](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request). This is how an inline comment should look like.
   
  image

## Optional suggestions

- [ ] I suggest using gifs through the code review will makes your review funny to a student.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by rivasbolinga** [comment]
Hi @Owusu-Desmond thank you for your review, I am afraid I am not getting exactly what I am doing wrong since I am adding inline descriptive comments to the code: I attach a screenshot of one of the comments. I wait your response, thanks in advance. 
Screenshot 2022-10-31 at 18 51 34

**Review by sajid-munawar — APPROVED** [review]
### Hi there,
Great job on this project review! 👍🏻
# Status: Approved ✔️
Excellent job on this project review. Keep it up 👏
Your project review is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉

### To Highlight
- [x]  Added descriptive inline comments. :heavy_check_mark: 
- [x]  Review comments are polite, use suggestions and give reasons for requested change :heavy_check_mark:
- [x] Made use of emojis in project review summary to make the code review more human-friendly 👽🤖. :heavy_check_mark: 
- [x] Made use of [GitHub inline comments feature](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request) :heavy_check_mark: 

 ## [Optional] suggestions:
 * Nothing to mention.

> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

> Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  FORCE_COLOR: 1
+
+jobs:
+  lighthouse:
+    name: Lighthouse
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
```
Comment (rivasbolinga) [inline .github/workflows/linters.yml:15]: ghjgjhfjgjgh

**Pull request merged**